rocAL/rocAL_pybind/examples/rocAL_api_python_unittest.py

```diff
@@ -27,20 +27,45 @@
 import amd.rocal.types as types
 from parse_config import parse_args
 import os
+import sys
+import cv2
 import cupy as cp
 
+INTERPOLATION_TYPES = {
+    0: types.NEAREST_NEIGHBOR_INTERPOLATION,
+    1: types.LINEAR_INTERPOLATION,
+    2: types.CUBIC_INTERPOLATION,
+    3: types.LANCZOS_INTERPOLATION,
+    4: types.GAUSSIAN_INTERPOLATION,
+    5: types.TRIANGULAR_INTERPOLATION
+}
+
+SCALING_MODES = {
+    0: types.SCALING_MODE_DEFAULT,
+    1: types.SCALING_MODE_STRETCH,
+    2: types.SCALING_MODE_NOT_SMALLER,
+    3: types.SCALING_MODE_NOT_LARGER
+}
+
+
 def draw_patches(img, idx, device):
-    #image is expected as a tensor, bboxes as numpy
-    import cv2
+    # image is expected as a tensor, bboxes as numpy
     args = parse_args()
     if device == "gpu":
         img = cp.asnumpy(img)
-    if not args.NHWC:
-        img = img.transpose([1, 2, 0])
     if args.fp16:
         img = (img).astype('uint8')
-    img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)
-    cv2.imwrite("OUTPUT_IMAGES_PYTHON/NEW_API/FILE_READER/" + args.augmentation_name + "/" + str(idx)+"_"+"train"+".png", img)
+    if not args.color_format:
+        img = img.transpose([0, 2, 3, 1])
+    images_list = []
```

Comment:
Whats this part of the code from line 60 - 62 doing ?

Where are we using image_list ?

Reply:
We stack the two images into a single image using cv.vconcat. It needs a list of np arrays for which this list is used to store the images

Reply:
So we are dumping it as one single image for muliple augmentations?

Reply:
No this script has no multiple augmentation test case at all. All outputs are for single image - exactly like the C++ unit tests